Problem2.rb

```diff
@@ -0,0 +1,31 @@
+# 2 largest numbers from integer array
+
+
+def TwoBiggestNumbers numbers
+
+
+
+   if numbers.nil?
+      raise ArgumentError, 'Argument is null' 
+   end   
+   if numbers.count < 2
+      raise ArgumentError, 'At least 2 elements in the array are required' 
+   end
+   numberA= numberB= numbers[0]

```

Comment:
What do `numberA` and `numberB` mean? 

The problem is about finding the 2 largest numbers, can you rename your variables so they make better sens for someone reading this code?